documentation/modules/auxiliary/client/sms/send_text.md

```diff
@@ -0,0 +1,123 @@
+The auxiliary/client/sms/send_text module allows you to send a malicious text/link to a collection
+of phone numbers of the same carrier.
+
+In order to use this module, you must set up your own SMTP server to deliver messages. Popular
+mail services such as Gmail, Yahoo, Live should work fine.
+
+## Module Options
+
+**CELLNUMBERS**
+
+The phone number (or numbers) you want to send the text to. If you wish to target against multiple
+phone numbers, ideally you want to create the list in a text file (one number per line), and then
+load the CELLNUMBERS option like this:
+
+```
+set CELLNUMBER file:///tmp/att_phone_numbers.txt
+```
+
+Remember that these phone numbers must be the same carrier.
+
+**SMSCARRIER**
+
+The carrier that the targeted numbers use. See **Supported Carrier Gateways** to learn more about
+supported carriers.
+
+**SMSMESSAGE**
+
+The text message you want to send.
+
+**SMTPADDRESS**
+
+The mail server address you wish to use to send the text messages.
+
+**SMTPPORT**
+
+The mail server port. By default, this is 25.
+
+**SMTPUSERNAME**
+
+The username you use to log into the SMTP server.
+
+**SMTPPASSWORD**
+
+The password you use to log into the SMTP server.
+
+**SMTPFROM**
+
+The FROM field of SMTP.
+
+## Supported Carrier Gateways
+
+The module supports the following carriers:
+
+* AllTel
+* AT&T Wireless
+* Boost Mobile
+* Cricket Wireless
+* T-Mobile
+* Verizon
+* Virgin Mobile
+* Google Fi
+
```

Comment:
With sprint being broken, instead of hiding it, i think it should be included in the docs to prevent 'what about sprint' tickets.
I'd make a new heading:

The module does not support the following carriers:

* Sprint: no valid gateways could be identified.

Will give some clues so hopefully a thoughtful ticket will appear saying "it works with foobar.baz" and boom, its working!